docs/Refund.md

```diff
@@ -6,28 +6,64 @@ If you are looking for information about requesting a refund from a merchant, pl
 
 **Refunds** are one of the applications built on top of the [Pull Payments](./PullPayments.md) feature.
 
-Up until now, merchants had to manually issue refunds.
-The process required a lot of back and forth between a sender and a receiver. The merchant had to ask for a refund address, calculate the preferred rate, manually add that to the wallet and broadcast the transaction.
-
-A merchant just needs to go to the invoice, click **Issue refund** , select the appropriate rate and share a link with a customer.
-
-The customer will then input their address and claim a refund. Once a customer claims a refund, the merchant will get a notification and can authorize it in a click.
-Future versions of BTCPay Server will enable instant payouts as an option, without the need for merchant approval.
+On this page, we will talk you through the flow of creating Refunds. 
+There are a short few steps to create the Refund for the customer. 
+After the refund has been issued, we will continue to the [Payouts](./Payouts.md) page to talk about the refund processing. 
 
 ## Create a refund
 
-1. After an invoice has been confirmed, go in the Invoices page and click `Details` on the invoice.
+1. After an invoice has been confirmed but needs a refund, go in the Invoices page and click `Details` on the invoice.
 
-![BTCPay Server refund feature](./img/pull-payments/9.png "BTCPay Server refund feature")
+![BTCPay Server refund feature](./img/refunds/invoices-details.jpg "BTCPay Server refund feature")
 
 2. Click `Issue a refund`
 
-![BTCPay Server refund feature](./img/pull-payments/10.png "BTCPay Server refund feature")
+![BTCPay Server refund feature](./img/refunds/issue-refund.jpg "BTCPay Server refund feature")
+
+3. Select the way how you want to issue the refund
+
+![BTCPay Server refund feature](./img/refunds/issue-refund-payment-option.jpg "BTCPay Server refund feature")
+
+4. Select the `amount` you want to refund
```

Comment:
Unrelated to this PR, but it would be good to discuss how to improve the UX of this page so people can actually understand it. This probably incldues better copyrighting.
![Screenshot 2022-04-01 at 11 30 01](https://user-images.githubusercontent.com/36959754/161236691-5bb02109-7074-4464-a039-13b35747f56d.png)